examples/lib/defaultHtmlStepUi.js

```diff
@@ -273,6 +275,25 @@ function DefaultHtmlStepUi(_sequencer, options) {
     }
   }
 
+  function imageHover(step){
+
+    var img = $(step.imgElement);
+
+    img.mousemove(function(e) { 
+      var canvas = document.createElement('canvas');
+      canvas.width = img.width();
+      canvas.height = img.height();
+      var context = canvas.getContext('2d');
+      context.drawImage(this,0,0);
+
+      var offset = $(this).offset();
+      var xPos = e.pageX - offset.left;
+      var yPos = e.pageY - offset.top;
```

Comment:
All this required an event for the position and that's why I added "mousemove" here and yess I did do it for onComplete and it works but I thought that instead on adding the code there and increasing it lets make it modular so I made a separate function.